sunpy_sphinx_theme/conf.py

```diff
@@ -21,24 +23,14 @@ def page_url(page):
     return urljoin(sunpy_website_url_base, page)
 
 
-def get_html_theme_path():
-    """
-    Return list of HTML theme paths.
-    """
-    theme_path = Path(__file__).parent.resolve()
-    return [str(theme_path)]
-
-
 html_theme = "sunpy"
-html_theme_path = get_html_theme_path()
-templates_path = [os.path.join(html_theme_path[0], html_theme, "templates")]
-html_static_path = [os.path.join(html_theme_path[0], html_theme, "static")]
-html_css_files = [os.path.join(html_static_path[0], "sunpy_style.css")]
-html_extra_path = [os.path.join(html_static_path[0], "img")]
-html_favicon = os.path.join(html_extra_path[0], "favicon-32.ico")
```

Comment:
I am concerned I am going to have broken something here, local testing on sunpy.org and the ndcube docs indicates it should be ok, but maybe we should do an rc and test it?